src/components/Match/MatchStory.jsx

```diff
@@ -562,6 +625,21 @@ const generateStory = (match) => {
     }
   }
 
+  // Remove any unneeded Time Markers
```

Comment:
Yeah, but what that part does is remove any markers that come right after another marker. That way if nothing happens between minute 30 and minute 40, instead of this:
**20min Marker**
event1
event2
**30min Marker**
**40min Marker**
event3
event4
event5

you get this:
**20min Marker**
event1
event2
**40min Marker**
event3
event4
event5